frontend/src/components/evaluationForm/EvaluationForm.tsx

```diff
@@ -25,7 +25,22 @@ export const EvaluationForm: React.FC = () => {
      * Qui devi implementare l'invocazione dell'api /api/calculate per ottenere il voto suggerito
      * Poi togli lo stub di codice qui sotto
      */
-    setSuggestedVote(9);
+
+    if (state.task_svolti_correttamente.fork_commit_pr) {
```

Comment:
- In react non dovresti aggiornare lo stato in modo incrementale con questa sintassi, dovresti usare setState((current) => newState)
- il task richiedeva di invocare l'api /calculate, che hai implementato ma che non invochi mai